test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -313,6 +326,26 @@ kv_workload_generator::choose_table(kv_workload_sequence_ptr txn)
     return _tables_list[_random.next_index(_tables_list.size())];
 }
 
+/*
+ * kv_workload_generator::generate_rand_stress_configs --
+ *     Generate random WiredTiger timing stress configurations.
+ */
+std::string
+kv_workload_generator::generate_rand_stress_configs()
+{
+    std::string wt_env_config = std::string(ENV_CONFIG);
+    probability_switch(_random.next_float())
+    {
+        probability_case(_spec.timing_stress_ckpt_slow) wt_env_config +=
+          ",timing_stress_for_test=[checkpoint_slow]";
+        probability_case(_spec.timing_stress_commit_txn_slow) wt_env_config +=
+          ",timing_stress_for_test=[commit_transaction_slow]";
+        probability_default wt_env_config =
```

Comment:
Doesn't this add by default `commit_transaction_slow`? I feel we should keep the default with no change.

Reply:
Fixed in b15a4f491906d1b92d4955cb07a321633d9f7722